src/api/app/views/webui2/webui/project/_project_packages.html.haml

```diff
@@ -0,0 +1,29 @@
+.card-body
+  - if packages.present?
+    %table.table.table-striped.table-bordered#packages-table
+      %thead
+        %tr
+          %th Name
+          %th Changed
+      %tbody
+        - packages.each do |package|
+          %tr
+            %td
```

Comment:
Same as above, I think this column should take more space (as some packages have long names). Perhaps 75% of the space available in the table.